as-derivative-script/kusama_derived_details.csv

```diff
@@ -0,0 +1,114 @@
+block,extrinsicIndex,signer,derivativeIndex,derivedAccount,success
+4801557,1,DqpNAWBTgPQFETepyPvigwrY6eTxtKS6obajvai8h7G18pn,61880,FmVgo9tXrw7Td4udEMCygPMwnWFsn3fPzgyGpFL7L6sFkcN,false
```

Comment:
Looks like some people do use values >255

Reply:
yea glad it's only a u16 🙈